docs/_docs/windows.md

```diff
@@ -17,45 +17,45 @@ If you are using Windows 10 Anniversary Update, the easiest way to run Jekyll is
 
 First let's make sure all our packages / repositories are up to date. Open a new Command Prompt instance, and type the following:
 
-```
+```shell
```

Comment:
can these be changed to <code>\`\`\`sh</code> please..? I don't think the new `TERMINAL` styles get applied to `shell` ...

Reply:
Now it does.  Stuck to ````sh` for consistency, but we're bulletproof.